Signal/test/util/SearcherTest.swift

```diff
@@ -244,16 +230,16 @@ class ConversationSearcherTest: XCTestCase {
         XCTAssertEqual([bookClubThread, snackClubThread, aliceThread], threads)
 
         threads = searchConversations(searchText: "Bob")
-        XCTAssertEqual(2, threads.count)
-        XCTAssertEqual([bookClubThread, bobThread], threads)
+        XCTAssertEqual(1, threads.count)
```

Comment:
So this is just a test change - not a feature change. I previously failed to update this test once we started intentionally excluding contact threads which had never been messaged from the "Conversations" search results.